### DISCUSSION TITLE
How do I turn off Keystone sound effects?

### DISCUSSION THREAD

**Discussion by WYCCW** [seerge/g-helper#3535] [Q&A]
I am using the ROG Strix G533ZW which comes with Keystone. But I don't know why it keeps ringing all the time. I know Armoury Create can  turn off it, but I don't want to use it. How do I turn off Keystone's sound effects via Ghelper?

**Comment by seerge** [comment]
@WYCCW hello,

Where does the sound come from ? If from the laptop speakers, most likely it's just a software thing from Armoury Crate itself. I.e. some leftover from AC keeps running and making that sound :) Did you uninstall AC using official AC uninstaller ?

**Reply by WYCCW** [reply]
The sound comes out of the speakers, accompanied by a light effect when the Keystone is plugged in, which is no different from the normal effect of unplugging it and plugging it back into the Keystone. I'm assuming that the recurring sound and light effects are due to poor contact, but turning off the plug-in sound and plug-in light effects through software should work, so I'm asking how to turn them off through Ghelper?
I uninstalled AC via the official software and I made sure it was completely uninstalled on my computer.

**Reply by seerge** [reply]
@WYCCW what I'm trying to say - that this is probably some other app from asus still running, that makes that sound :) So if you will find and stop it - sound will stop as well? Check windows Task Manager for everything "asus".

**Reply by WYCCW** [reply]
@seerge 
Happy New Year!
I closed all asus processes in task manager and re-plugged Keystone, but the sound effect is still there, how can the problem be solved?

**Reply by seerge** [reply]
@WYCCW does the sound / effect play when you are not in windows ? For example when you are booting or in BIOS menu (F2 on boot) ?

If so - it could be some firmware setting responsible for that. May be try to the toggle in BIOS menu -> Advanced Settings then ?

**Reply by WYCCW** [reply]
@seerge The problem seems to have been solved, I deleted "armourycreatecontrolinterface.exe" from my computer and it seems to be working fine, now even if I plug and unplug the Keystone normally I don't get the lights and sounds, even though this was the original setting of the asus at the time of design. Now no matter what I do no sound or light effects are produced.

Another thing I would like to ask is that every time I turn on my computer, ghelper now says "ghelper is running", which is a bit annoying, how do I disable this box?

**Reply by seerge** [reply]
@WYCCW 

So it was asus process actually doing that ?

As for startup - it looks like you have scheduled G-Helper twice under different accounts ? Just launch app at least once as admin, and then uncheck / check `Run on Startup` checkbox. It would recreate startup task again.

**Reply by WYCCW** [reply]
@seerge I can't say for sure that it's the asus program that's doing this, but I suspect it is

BTW, with your help, the problem of "ghelper is running" popping up every time I turn on my computer has been solved, thank you again for all the help you've provided 🙏🙏

**Reply by WYCCW** [reply]
@seerge 

Hello Seerge,

Previously, I discussed with you how to turn off the lighting effects and sounds of the Keystone. I mentioned that I tried deleting "armourycreatecontrolinterface.exe," which successfully turned off the lighting and sound of the Keystone, and I thought this was the solution. However, while this did work for a while, the lighting and sound started to appear again shortly after. I tried deleting "armourycreatecontrolinterface.exe" again, but they still persisted. I realized that my conclusion might have been incorrect, and I apologize if this caused any confusion or negatively affected your judgment and led to undesirable consequences.

At the time of my inquiry, Armoury Create was not installed on my computer, so I decided to download and install it. Then, within Armoury Create, I turned off the lighting and sound for the Keystone. Afterward, I uninstalled the software, and now I’ve achieved the desired result: no lighting or sound effects, whether I plug in or unplug the Keystone. I’m very satisfied with this outcome.

Once again, I apologize if my incorrect conclusion affected your judgment and development.🙏🙏

**Reply by seerge** [reply]
@WYCCW hello, no worries. I'm glad it's resolved for you
